core/utils/poly-look/src/visualisations/charts/bar-charts/horizontalBarChart/horizontalBarChart.js

```diff
@@ -237,13 +237,16 @@ export class HorizontalBarChart extends Chart {
       .attr("fill", this._barValueColor);
   }
 
-  _updateExistingBarValues(barValues) {
-    barValues
+  _updateExistingBarValues(barGroups) {
+    barGroups
```

Comment:
Here you're just renaming a variable for some reason, shouldn't change anything. So that's not the fix, right?